conf/machine/olinuxino-a20lime2-emmc.conf

```diff
@@ -0,0 +1,12 @@
+#@TYPE: Machine
+#@NAME: Olimex A20-OLinuXino Lime2 Board with 4GB eMMC
+#@DESCRIPTION: Machine configuration for the Olimex A20-OLinuXino Lime2 Board with 4GB eMMC, based on Allwinner A20 CPU
+#https://github.com/OLIMEX/OLINUXINO
+
+require conf/machine/include/sun7i.inc
+
+PREFERRED_VERSION_linux = "4.11.0+git%"
```

Comment:
I think it's better to keep kernel version consistent with non-emmc board, it works fine with the default 4.9.x kernel so please remove this